unipipe/blueprints/basic-handler.js.js

```diff
@@ -38,7 +40,7 @@ class MyHandler {
 }
 
 const handlers = {
-  "d90c2b20-1d24-4592-88e7-6ab5eb147925": new MyHandler(),
+  "${uuid.generate()}": new MyHandler(),
```

Comment:
I would advise against doing this, as this will prevent the catalog entry to match the handler entry.

Reply:
but the whole catalog is already using auto generated uuids. so i actually steal this idea from it =) . i actually added a comment line on terraform-handler. maybe i can add that same comment over here as well

Reply:
Good point - then pointing it out with comments is fine. 

The alternative would be using a fixed uuid for the matching entry in the catalog and the handler, so everything for this works "out of the box", without fiddling around in the handler file. But using randomly generated uuids prevents id collisions in test-marketplace scenario which is also important, so let's go with what you suggested.

Reply:
one additional alternative I see would be to prompt for a UUID when generating the basic handler example, the prompt could say "1: give me service definition UUID, 2: just use a random one (default)"

Reply:
done. i did this with old school placeholder method 👍